### ISSUE TITLE
export Image

### ISSUE THREAD

**Issue by jsonseng** [akisute/AKWebRenderer#1]
Hi, I have try to use method(__cachedSnapshotViewForRenderRequest) to get the copy view(mark A).

But when I call method(renderInContext) from the copy view(mark A) still get a blank picture

**Comment by akisute** [comment]
@jsonseng Sorry for being so late for reply! I'm in summer vacation right now :(

Anyway which version of iOS have you tried? This experimental code is rather old and I have only confirmed everything in iOS 8 (possibly in iOS 9 as well). So things might have been changed since then...

**Comment by jsonseng** [comment]
I try in iOS9.
I have been use method(drawViewHierarchyInRect) in the copy view getting image.But only work well in simulator. In the device, I still get blank image.

Thank you for all your assistance.

**Comment by akisute** [comment]
@jsonseng OK, I have tried the code myself on iOS 9.3.1 and confirmed that the workaround is no longer working on iOS 9. iOS 9 still returns `_UIReplicantView`, `_UIReplicantContentView` and `CASlotProxy` when `resizableSnapshotView(from rect: CGRect, afterScreenUpdates afterUpdates: Bool, withCapInsets capInsets: UIEdgeInsets) -> UIView?` is called, though. Perhaps they _fixed_ this issue after all... I can't find the another workaround to make this work after several different attempts. I'm going to update the documentation to clarify this.

Sorry for inconvenience.

**Comment by jsonseng** [comment]
Thank you very much. I have tried serveral ways but still not work. only the way using presentationLayer(renderInContext:) can get the correct image. But No speed. I think If I have to use (drawViewHierarchyInRect) to get the image. I should change the view without using CATiledLayer. 

I really appreciate that

**Comment by akisute** [comment]
Yeah that private method `UIView. drawViewHierarchyInRect()` might actually work! I'm not entirely sure the method is still available in recent iOS (since I've stopped using that after iOS 8), but it's seemed to be worth trying :)

**Comment by jsonseng** [comment]
It can be work in a simple layer(CALayer) no matter the device or simulator. If the view base on CATiledLayer(or some special layer), `drawViewHierarchyInRect()` still get a blank image in device, but work in simulator.

**Comment by akisute** [comment]
Ugh, looks like a terrible issue 😭 but now I guess all source of this blank image bug could be CATiledLayer (of UIWebView, in this case). If you somehow stop using CATiledLayer and use a basic CALayer instead, my workaround code might possibly still work.

**Comment by jsonseng** [comment]
I maybe try to override the CATiledLayer Instead of using system.  If it is useful for your project, when I finish it, i send the framework to you.
